web/static/js/store/configureStore.dev.js

```diff
@@ -2,5 +2,5 @@ import prodStore from './configureStore.prod'
 import createLogger from 'redux-logger'
 
 export default function configureStore(preloadedState) {
-  return prodStore(preloadedState, [createLogger()])
+  return prodStore(preloadedState, [createLogger({ collapsed: true })])
```

Comment:
Do we need to do the same at `web/static/mobile_survey/js/store/configureStore.dev.js`?

Reply:
Not needed but :+1: